src/main/proto/vector_service.proto

```diff
@@ -137,7 +137,9 @@ message QueryRequest {
   bool                   include_metadata = 5;
 
   // The query vectors.
-  repeated QueryVector   queries = 6;
+  repeated QueryVector   queries = 6 [
+  deprecated = true
+  ];
```

Comment:
is this also deprecated in the `vector_service.proto` file in pinecone-db and pinecone-protos repos?

Reply:
Yes!